src/main/java/net/minecraftforge/client/SkyRenderHandler.java

```diff
@@ -53,19 +55,33 @@
     }};
 
     /**
-     * Registers sky renderer for certain pass and id.
-     * Registering a renderer with pre-existing id will replace the previous renderer.
-     * Renderer with higher priority is called earlier.
+     * <p>Registers sky renderer for certain pass and id.</p>
+     * <p>Registering a renderer with pre-existing pass & id pair will replace the previous renderer.</p>
+     * <p>You can unregister a renderer by giving <code>null</code> for the renderer parameter.
+     * Unregistering a renderer with its vanilla-counterpart will re-enable a vanilla renderer.</p>
+     * <p>Renderer with higher priority is called earlier.</p>
      * @param pass the sky render pass
      * @param id the id for the renderer
      * @param renderer the renderer
      * @param priority the priority for the renderer
      * */
-    public void registerSkyRenderer(SkyRenderPass pass, ResourceLocation id, IRenderHandler renderer, double priority)
+    public void registerSkyRenderer(SkyRenderPass pass, ResourceLocation id, @Nullable IRenderHandler renderer, double priority)
     {
         renderers.get(pass).registerSkyRenderer(id, renderer, priority);
     }
 
+    /**
+     * Unregisters vanilla sky renderer for certain pass and id.
+     * This will only remove the renderer if it's the vanilla one.
+     * @param pass the sky render pass
+     * @param id the id to unregister
+     * @return the vanilla render handler, or <code>null</code> if there wasn't a vanilla renderer for specified pass-id pair
+     * */
+    public @Nullable IRenderHandler unregisterVanillaRenderer(SkyRenderPass pass, ResourceLocation id)
```

Comment:
If there's only 1 vanilla renderer for each pass, does it make sense for the `id` to be passed in?

Reply:
Ah, I see. Whoops.